message/router.go

```diff
@@ -219,6 +275,41 @@ func (r *Router) Run() (err error) {
 		}
 	}
 
+	r.logger.Debug("Applying decorators", nil)
```

Comment:
it's looooong, please clean up it in a wise way (not just to one func pls) because router is potentially very complex part

Reply:
9bf86d0855a0af420e5271cb9079119fe7f16e66 i split this whole fragment into `func (r *Router) decorateHandlerPublisher(h *handler) error` and `func (r *Router) decorateHandlerSubscriber(h *handler) error`